test/e2e/logic-function.e2e.js

```diff
@@ -0,0 +1,227 @@
+const { expect } = require('../setup');
```

Comment:
In general this test might fail on concurrent executions especially the `list` one